src/uv-config.json

```diff
@@ -3,12 +3,40 @@
         "rightPanelEnabled": false
     },
     "modules": {
-        "footerPanel": {
-            "options": {
-                "downloadEnabled": false,
-                "fullscreenEnabled": true,
-                "shareEnabled": false
-            }
+      "headerPanel": {
```

Comment:
Do we need this section? It looks like this is the UV default.

Reply:
I removed the optional code. The lines `moreInfoEnabled` and `shareEnabled` default to `true` so setting them `false` needs to be retained.